ultimatejobweb/admin.py

```diff
@@ -1,3 +1,14 @@
-# from django.contrib import admin
+from django.contrib import admin
+from .models import user, company
+from django.apps import apps
 
 # Register your models here.
+admin.site.register(user)
+admin.site.register(company)
```

Comment:
why manually register user and company, if they'll get registered automatically in the code below?

Reply:
because they already exist. I have to so it that way.
I tried without it and it doesn't work

Reply:
The code below is supposed to register all the models including the ones above.
If that doesn't work, then there is no point in the code below at all.